javascript/ui/element/name.js

```diff
@@ -76,13 +76,15 @@ element.name.validate_ = function(nameElement, errorElement) {
 element.name.initNameElement = function() {
   var nameElement = element.name.getNameElement.call(this);
   var errorElement = element.name.getNameErrorElement.call(this);
-  element.listenForInputEvent(this, nameElement, function(e) {
-    // Clear but not show error on-the-fly.
-    if (element.isShown(errorElement)) {
-      element.setValid(nameElement, true);
-      element.hide(errorElement);
-    }
-  });
+  if (nameElement && errorElement) {
```

Comment:
I mention this logic in a comment below. I don't think it should be changed here.